test/test_user_commons/test_labels.py

```diff
@@ -0,0 +1,70 @@
+from unittest import TestCase
```

Comment:
you should create e2e test in `test_dataset_manager` and test on real table in bigquery